_posts/2017/2017-02-23-scrum-interview.md

```diff
@@ -0,0 +1,140 @@
+I work very closely with the developer teams.
+So, I am really a part of each team.
+With every team, we always look at the processes and how we can improve.
+Together, we estimate the scope of stories and if something remains unclear we conduct little amends.
+Working with Scrum implies a regularity in meet-ups and I am responsible to moderate these meetings.
+Therefore, we have a so-called daily, in which the developers give a brief update on what they are currently working on and whether there are any impediments whatsoever.
+Sometimes, I need to stop upcoming discussions and postpone them to after the daily to not lose focus - I like to always keep the daily short and informative.
+
+Also, I collaborate closely with the Product Owner and pass on his feedback to the teams.
+Together, we check the backlog and make sure that the meetings are scheduled in a way that everybody can attend.
+I am also the link between development and management.
+You see, I keep up the communication between all parties - CTO, Product Owners, and developers.
+Beyond that, I have other exciting tasks, such as organizing the yearly Hackathon and R&D day.
+Or just in general, I organize team events, because sometimes it is simply nice to go out and have fun.
+
+### Are there already any insights on this year's Hackathon you could reveal?
+
+We are organizing the Hackathon for the end of March and of course it's going to be lots of fun {% emoji wink %}.
+It'll be one day and a half and we just started the idea-finding process.
+Our developers can formulate some wishes, we then go over the list and see which topics will make it.
+I think it's just going to be fun, because everyone is pursuing his hacking passion and of course I will order enough pizza and have enough coffee ready, so the developers are completely fueled.
+At the end of the Hackathon, we will then have some drinks to celebrate ourselves.
+
+### Very briefly, explain us your work in a Scrum team. How is it organized with regards to meetings, moderation, etc.?
+
+Some teams have two week sprints.
+I will make sure that at the beginning of a sprint, we will have our sprint planning and have the backlog prepared for this meeting.
+We go through it and schedule topics for the upcoming sprint.
+I also ensure that the team doesn't over or underestimate what they can achieve within one sprint.
+When we start our sprints, we of course have our mentioned dailies every morning.
+In the middle of the sprint, we have a refinement, in which we go through the tickets in the backlog and discuss and estimate them in order to have them ready for the next sprint planning, which then happens a week later.
+At the end of the sprint, we have a sprint review, which is open for everyone in the company who is interested.
+
+For that, we make sure that we have a brief presentation, running software we can show and that we get an invite, an internal blog post, out beforehand so everyone knows when and where the meeting takes place.
```

Comment:
For the sprint review meeting, the team prepares a brief presentation, running software we can show and send out an invite so that everyone knows beforehand when and where the meeting takes place.